ethcore/evm/src/interpreter/gasometer.rs

```diff
@@ -32,6 +31,10 @@ macro_rules! overflowing {
 	}}
 }
 
+lazy_static! {
+	static ref PRECOMPILES_ADDRESS_LIMIT: Address = Address::from([0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0xff,0xff]);
```

Comment:
can it be a const?

Reply:
`From::from` is not a `const fn`

Reply:
As a side note, we should eventually replace `lazy_static` with `once_cell` as a less magical, more modern alternative.

Reply:
The compiler complains `found type alias Address: can't use a type alias as a constructor`.
I checked the code, and I found that is a common pattern that Address constants are initialized with `lazy_static`.

Reply:
> The compiler complains found type alias Address: can't use a type alias as a constructor.

Ah, this E0423 is annoying, and it actually sounds like a compiler bug to me.
We could use something like this as workaround
```rust
const LIMIT: Address = H160([...]);
```
As a side note, we should replace `lazy_static` with consts wherever possible, now that more const functions features are getting stable.

Reply:
fixed in 21bf129